### PR TITLE
Normal map support on translucent objects

### PR DESCRIPTION
**Pull request by Zvork**
As the title states, this PR adds support for normal maps on translucent objects. Please note, that the PR also encompasses changes proposed in PR #12180 .

# Test cases
A _hifi_tests/tests/render/engine/material_ update, especially the "opacity" test, for this is proposed in [hifi_tests PR #68](https://github.com/highfidelity/hifi_tests/pull/68). Unfortunately, due to a known bug, it is sometimes necessary to reload the "opacity.js" test script multiple times to see the normal map applied.

It is also suggested to test with any custom made assets.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6590/app_12255_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6590/HighFidelity-Beta-PR12255-d0f2138e8785f6f9dcac5a822eb8695bd3fcc11e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6590/HighFidelity-Beta-PR12255-d0f2138e8785f6f9dcac5a822eb8695bd3fcc11e.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6666/app_12255_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6666/HighFidelity-Beta-PR12255-70a7ecc5f06dccad36f87f135589cdfeccf990cf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6666/HighFidelity-Beta-PR12255-70a7ecc5f06dccad36f87f135589cdfeccf990cf.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6860/app_12255_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6860/HighFidelity-Beta-PR12255-a44c5c9814af6e7d572837b1bad99f5aeebe1e21.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6860/HighFidelity-Beta-PR12255-a44c5c9814af6e7d572837b1bad99f5aeebe1e21.exe

File: libraries/render-utils/src/RenderPipelines.cpp [hunk 1]
```diff
@@ -72,13 +72,16 @@
 #include "model_lightmap_normal_specular_map_frag.h"
 #include "model_lightmap_specular_map_frag.h"
 #include "model_translucent_frag.h"
+#include "model_translucent_normal_map_frag.h"
 #include "model_translucent_unlit_frag.h"
+#include "model_translucent_normal_map_frag.h"
```
Comment (samcake) [inline libraries/render-utils/src/RenderPipelines.cpp]: DIdn;t you include twice the same here ?

Comment (Zvork) [reply libraries/render-utils/src/RenderPipelines.cpp]: Absolutely... I oversaw that during the merge.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6869/app_12255_PR.apk

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6870/app_12255_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6870/HighFidelity-Beta-PR12255-03daba9bacad4b82a3e6e79625608bd23af1cc06.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6870/HighFidelity-Beta-PR12255-03daba9bacad4b82a3e6e79625608bd23af1cc06.exe

**Comment by Qliemillar** [comment]
Currently being tested

**Comment by Qliemillar** [comment]
@Zvork Hello, could we please get a little clarification on how to test? 

So far we've gone into several domains and looked at objects with transparent textures (glass, etc) as well as checked several avatars with transparent textures in their hair. So far all is looking well, but we want to make sure we're not missing anything.

Is our current testing sufficient, or is there anything else that should be checked?

**Comment by samcake** [comment]
Speaking for @zvork i think it s good enough.

**Comment by Qliemillar** [comment]
Alright cool! Then it's a pass! @conklin94122

**Pull request merged**